monkestation/code/modules/storytellers/converted_events/event_overrides.dm

```diff
@@ -29,21 +29,22 @@
 /datum/round_event_control/brand_intelligence
 	track = EVENT_TRACK_MODERATE
 	tags = list(TAG_DESTRUCTIVE, TAG_COMMUNAL)
-	shared_occurence_type = SHARED_BSOD
+	event_group = /datum/event_group/bsod
 
 /datum/round_event_control/bureaucratic_error
 	track = EVENT_TRACK_MAJOR // if you've ever dealt with 10 mimes you understand why.
 	tags = list(TAG_COMMUNAL)
-	shared_occurence_type = SHARED_BSOD
+	event_group = /datum/event_group/error
 
 /datum/round_event_control/camera_failure
 	track = EVENT_TRACK_MUNDANE
 	tags = list(TAG_COMMUNAL, TAG_SPOOKY)
-	shared_occurence_type = SHARED_BSOD
+	event_group = /datum/event_group/error
```

Comment:
dont put this in a group, its gonna make everything else in said group pretty much never run

Reply:
done